pkg/translator/prometheusremotewrite/otlp_to_openmetrics_metadata_test.go

```diff
@@ -167,11 +167,12 @@ func TestOtelMetricsToMetadata(t *testing.T) {
 					Type: prompb.MetricMetadata_GAUGE,
 					MetricFamilyName: func() string {
 						metricNamer := otlptranslator.MetricNamer{WithMetricSuffixes: false, Namespace: ""}
-						return metricNamer.Build(prom.TranslatorMetricFromOtelMetric(getIntGaugeMetric(
+						metricName, _ := metricNamer.Build(prom.TranslatorMetricFromOtelMetric(getIntGaugeMetric(
```

Comment:
I was going to say require noerror but that is a lot of places to do so. Probably, if there were any error some subsequent test would fail anyway? so it seems ok.

Reply:
Yep, that's my assumption as well